### DISCUSSION TITLE
Dataset Selection

### DISCUSSION THREAD

**Discussion by LTMeyer** [PolymathicAI/the_well#10] [Q&A] [answered]
How did you select the different datasets?

**Comment by mikemccabe210** [comment, marked as answer]
The selection process was application-driven. We reached out to computational scientists and numerical software developers and asked for either tasks that were relevant to their research or that they found to be challenging test cases for existing software. From there, since our goal was to develop a ML-accessible dataset, we filtered out cases where the primary challenge was scale or geometry or where there simply wasn’t enough data. To keep the focus on dynamics, we ended up selecting only systems that could be sampled reasonably on uniform space and time grids.

**Reply by vaibhavswire** [reply]
Add it all to Kaggle too, best wishes
